lib/knackhq/client.rb

```diff
@@ -0,0 +1,42 @@
+require 'knackhq/client/version'
+require 'blanket'
+require 'hashie'
+
+module Knackhq
+  class Client
+    attr_accessor :base_uri, :x_knack_application_id, :x_knack_rest_api_key
+
+    def initialize(base_uri, x_knack_application_id, x_knack_rest_api_key)
+      @base_uri = base_uri
+      @x_knack_application_id = x_knack_application_id
+      @x_knack_rest_api_key = x_knack_rest_api_key
+    end
+
+    def objects
```

Comment:
So in this case, it would look like:
```Ruby
+module Knackhq
+  class ClientQuery
# objects as results
# similar to the Blanket Wrapper gem
  end
end

ClientQuery.object_1
# returns object_1 attributes

ClientQuery.objects
# returns all objects
```